www/assessment/activities-i18n.php

```diff
@@ -12,7 +12,17 @@
 use LearnositySdk\Request\Init;
 use LearnositySdk\Utils\Uuid;
 
-$language = filter_input(INPUT_GET, 'language', FILTER_SANITIZE_FULL_SPECIAL_CHARS, ['options' => ['default' => 'en-US']]);
+function validateLanguageParam(string $language)
+{
+    $pattern = '/^[a-z]{2}-[A-Z]{2}$/';
```

Comment:
No sure if there is a strict guarantee that we'll only have two letters on each side. OTOH, I don't know of a counter example.